### PR TITLE
Feat/signal

### PR DESCRIPTION
**Pull request by AliMD**

### CODE REVIEW

**Comment by AliMD** [comment]
@asmanp2012

File: package/signal/src/type.ts [hunk 1]
```diff
@@ -0,0 +1,68 @@
+declare global
+{
+  /**
+   * Global signal registry.
+   */
+  interface VatrSignals
```
Comment (asmanp2012) [inline package/signal/src/type.ts:6]: Please change vatarSignals to signalList

File: package/signal/src/type.ts [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+  {
+    readonly 'easter-egg': void;
+  }
+
+  interface VatrRequestSignals
```
Comment (asmanp2012) [inline package/signal/src/type.ts:11]: Please change RequestSignals to RequestSignalList

**Review by asmanp2012 — CHANGES_REQUESTED** [review]
Please chnage signals to signalList

**Review by asmanp2012 — APPROVED** [review]
Everything is ok.

**Pull request merged**